coeus/coeus_analysis/src/analysis/instruction_flow.rs

```diff
@@ -1734,6 +1734,12 @@ impl InstructionFlow {
                 }
                 Instruction::ArrayData(_, _) => {}
                 Instruction::SwitchData(_) => {}
+
+                Instruction::ShrIntLit8(dst, left, lit) => {
+                    b.state.registers[u8::from(dst) as usize] =
+                        &b.state.registers[u8::from(left) as usize].wrapping_shr(lit as i128)
```

Comment:
```suggestion
                        &b.state.registers[u8::from(left) as usize] >> (lit as i128)
```
Sorry didn't test/realize that this is a `Value`, and we only have Shl/Shr implemented.